rest-service/manager_rest/storage/storage_manager.py

```diff
@@ -629,3 +673,95 @@ def __iter__(self):
 
     def __getitem__(self, item):
         return self.items[item]
+
+
+class SQLAlchemyTracingHelper(object):
+    @staticmethod
+    def enable_tracing_for_all_engines():
+        listen(Engine, 'before_cursor_execute',
+               SQLAlchemyTracingHelper._engine_before_cursor_handler)
+        listen(Engine, 'after_cursor_execute',
+               SQLAlchemyTracingHelper._engine_after_cursor_handler)
+        listen(Engine, 'handle_error',
+               SQLAlchemyTracingHelper._engine_error_handler)
+
+    @staticmethod
+    def _clear_traced(obj):
+        '''
+        Clear an object's decorated tracing fields,
+        to prevent unintended further tracing.
+        '''
+        if hasattr(obj, '_parent_span'):
+            del obj._parent_span
+        if hasattr(obj, '_traced'):
+            del obj._traced
+
+    @staticmethod
+    def _normalize_stmt(statement):
```

Comment:
Explain what is normal and why this is needed?

Reply:
I looked over the implementation of the [sqlalchemy wrapper](https://github.com/opentracing-contrib/python-sqlalchemy/blob/master/sqlalchemy_opentracing/__init__.py#L138) and they had it. I guess this is because otherwise you'd get a bloated text when you view the query.

Reply:
I reviewed this once again and I think without normalization it's much more readable